components/company/languages_badges.tsx

```
@@ -0,0 +1,24 @@
+"use client";
+import { Badge } from "@/components/ui/badge"
+import { useState } from "react";
+
+type Language = {
+    languages: string[]
+}
+
+
+export default function languagesBadges({ languages }: Language) {
+    const [programmingLanguages, setProgrammingLanguages] = useState([]);
```

ESLint

[error] 'programmingLanguages' is assigned a value but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] 'setProgrammingLanguages' is assigned a value but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] React Hook "useState" is called in function "languagesBadges" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)